tahowallet/extension
# Disable network fees for custom networks.

Closes https://github.com/tahowallet/extension/issues/3189

<img width="387" alt="Screen Shot 2023-03-28 at 11 08 09 AM" src="https://user-images.githubusercontent.com/94649004/228316194-19be410c-8cc4-48e1-a28b-ef084f5d4ee8.png">

### To Test
- Add the Evmos chain to the wallet
- Attempt to send EVMOS to an address
- Notice that gas settings are greyed out with the correct tooltip.

Latest build: [extension-builds-3203](https://github.com/tahowallet/extension/suites/12040401263/artifacts/633008979) (as of Wed, 05 Apr 2023 08:40:51 GMT).
